harness/tests/test/unit/test_calls.gd

```diff
@@ -11,6 +11,49 @@ func test_godot_to_jvm_call_using_new() -> void:
 	assert_eq(invocation_script.int_addition(2, 7), 9, "call to addition jvm method should return 2 + 7")
 	invocation_script.free()
 
+func test_get_int_call_using_scene() -> void:
+	var invocation_scene = load("res://Spatial.tscn").instance()
```

Comment:
maybe use `_new()` instead of loading the scene?

Reply:
I have two test, one with scene, the other with new :P

Reply:
I think the scene one is redundant, it's not really testing our code.

Reply:
yep that's true, new already test instance_create